<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDialog</class>
 <widget class="QDialog" name="CreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hlayout_0">
     <item>
      <widget class="QLabel" name="label_type_0">
       <property name="text">
        <string>Строковый</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_0">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_0">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_0">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_1">
     <item>
      <widget class="QLabel" name="label_type_1">
       <property name="text">
        <string>Числовой</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_1">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_2">
     <item>
      <widget class="QLabel" name="label_type_2">
       <property name="text">
        <string>Числовой (с рисунком)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_2">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_3">
     <item>
      <widget class="QLabel" name="label_type_3">
       <property name="text">
        <string>Соответсвие</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_3">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_4">
     <item>
      <widget class="QLabel" name="label_type_4">
       <property name="text">
        <string>Множественный</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_4">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_5">
     <item>
      <widget class="QLabel" name="label_type_5">
       <property name="text">
        <string>Последовательность</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_minus_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box_5">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_plus_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="check_box_lockal_base">
          <property name="text">
           <string>Локальная база</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butt_create">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butt_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
